inject/src/main/java/io/micronaut/inject/ast/ElementFactory.java

```diff
@@ -31,135 +30,115 @@
  * @since 2.3.0
  */
 public interface ElementFactory<E, C extends E, M extends E, F extends E> {
```

Comment:
A bit concerned about the breaking changes to this interface. Can we retain the original methods and add new methods? The old methods and create a factory that just delegates to the passed metadata.

Reply:
I had them before as deprecated, but we want to remove everything deprecated in the v4. So I deleted them.